code/game/objects/items/devices/magnetic_lock.dm

```diff
@@ -11,46 +11,51 @@
 	icon = 'icons/obj/magnetic_locks/centcom.dmi'
 	icon_state = "inactive"
 	w_class = 3
-	req_access = list(103)
+	req_access = list(access_cent_specops)
 	health = 90
 
 	var/department = "CENTCOM"
 	var/status = 0
+	var/locked = 0
+	var/hacked = 0
```

Comment:
You are using a mixture of bitflags (STATUS_ ) and binary vars to set the status of the device.
It would be cleaner if you choose one and stick to it.